### ISSUE TITLE
Build Fails 6.0.3

### ISSUE THREAD

**Issue by heakinakai** [elastic/kibana-plugin-helpers#38]
Hi, 

You've added an OS check into your create_build.js for win_cmd but this errors when you run the build task on OSX.

#37 

Error: BUILD ACTION FAILED: { Error: spawnSync yarn ENOENT

I've fixed this by changing the following lines, but you might want to change cmd back to 'npm' after the try.

```
try {
        // use yarn if yarn lockfile is found in the build
        cmd = winCmd('yarn');
        statSync(join(buildRoot, 'yarn.lock'));
        execFileSync(cmd, ['install', '--production'], options);
      } catch (e) {
        // use npm if there is no yarn lockfile in the build
        execFileSync('npm', ['install', '--production', '--no-bin-links'], options);
      }
```

I realise this works because I don't have a lock file yet and I've changed it back to 'npm'.. so it's not the solution..

maybe something like this in create_build.js would work:

```
} catch (e) {
// set cmd back to npm after try
        cmd = winCmd('npm');
        // use npm if there is no yarn lockfile in the build
        execFileSync(cmd, ['install', '--production', '--no-bin-links'], options);
      }
```

I don't have a windows machine to test any of this.

Anyone with this issue, rolling back to 6.0.2 fixed this for me.

**Comment by spalger** [comment]
Sorry about that, fixed in d48808f8d832bda1dd11f84724958ad78e08aa0e. Can you try 6.0.4?

**Comment by heakinakai** [comment]
Hi @spalger,

Great work, 6.0.4 is working.

closing the problem
